File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/php/patricioleono.php
<?php
$cantidad = 100;
for($i = 1; $i <= $cantidad; $i++){
    if ($i % 3 == 0 && $i % 5 == 0){
        echo 'fizzbuzz';
        echo "'\n'";
    }elseif ($i % 5 == 0) {
        echo 'buzz';
        echo "'\n'";
    }elseif($i % 3 == 0){
        echo 'fizz';
        echo "'\n'";
    }else {
        echo $i;
        echo "'\n'"; 
    }
}

?>

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/java/patricioleono.java
public class patricioleono {

    public static void main(String[] args){
     int total = 100;
     for(int i = 0; i <= total; i++){
         if(i % 2 == 0){
             System.out.println("Fizz");
             if(i % 3 == 0){
                 System.out.println("FizzBuzz");
             }
         }else if(i % 3 == 0){
             System.out.println("Buzz");
         }else{
             System.out.println(i);
         }
     }
   }

}
